cluster/etcd.go

```diff
@@ -68,6 +68,16 @@ func (c *Cluster) DeployRestoreCerts(ctx context.Context, clusterCerts map[strin
 }
 
 func (c *Cluster) DeployStateFile(ctx context.Context, stateFilePath, snapshotName string) error {
+	stateFileExists, err := util.IsFileExists(stateFilePath)
+	if !stateFileExists {
+		logrus.Warnf("Could not read cluster state file from [%s], file does not exist. Snapshot will be created without cluster state file. You can retrieve the cluster state file using 'rke util get-state-file'", stateFilePath)
+		return nil
+	}
+	if err != nil {
```

Comment:
Should check the error first. The `util.IsFileExists` always return false, err so if there is an error it won't show up.